src/packages/core/index.ts

```diff
@@ -1,11 +1,23 @@
 import { ConnectorsByName, DefaultFlavor, FlavorName } from "@ganache/flavors";
+import { KNOWN_CHAINIDS } from "@ganache/utils";
 import ConnectorLoader from "./src/connector-loader";
 import { ProviderOptions, ServerOptions } from "./src/options";
 import Server from "./src/server";
 export { Server, ServerStatus, _DefaultServerOptions } from "./src/server";
-
 export type { Provider, EthereumProvider, FilecoinProvider } from "@ganache/flavors";
 export type { ProviderOptions, ServerOptions } from "./src/options";
+export type _ExperimentalInfo = Readonly<{
+  version: string,
+  fork: Readonly<{
+    /**
+     * Chains fully supported by Ganache forking.
```

Comment:
By "Fully supported" I really mean "ethereumjs-vm claims to know the rules" and I just assume they get it right. We do nothing with the Chain ID other than pass it to ethereumjs-vm. :thinking:

Reply:
I see, that makes sense. In the future it would be really cool to have a test suite that "proves" "full support", or points to the differences between Ganache + some chain ID and the same transactions on that chain.

Maybe it spins up a test Ganache instance at block `n` of the chain we want to test, and has a reference instance of Ganache at block `n+1`. Then it runs the same transactions that were on the `+1` block onto the `n` block and finds the differences in the resulting block/transactions.

Reply:
Updated the code and comment and create a new issue: https://github.com/trufflesuite/ganache/issues/2560